devtools/conda-envs/test_env.yaml

```diff
@@ -0,0 +1,16 @@
+name: test
+channels:
+- conda-forge
+dependencies:
+- python
```

Comment:
Do we need to specific python here? Does this work with the Python matrix?

Reply:
It is here
[extra-specs: |
            python=${{ matrix.python-version }}](https://github.com/alchemistry/alchemlyb/pull/191/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR36)

Reply:
My English was bad: What I meant was "Do we need to include (=specify) python in this file at all as a dependency, given that we are already setting it in the build matrix?" 

Will the CI also work if there's no "python" inside test_env.yaml? But that's a minor thing, given that CI works at the moment.